pdns/lua-record.cc

```diff
@@ -729,6 +727,24 @@ static void setupLuaRecords()
       return std::string("unknown");
     });
 
+  lua.writeFunction("filterForward", [](string address, std::vector<std::pair<int, string>> masks) {
+      NetmaskGroup nmg;
+      ComboAddress ca(address);
+
+      for (const auto& mask : masks) {
+        nmg.addMask(mask.second);
+      }
+      if (nmg.match(ComboAddress(address))) {
+        return address;
+      } else {
+        if (ca.isIPv4()) {
+          return string("0.0.0.0");
```

Comment:
Would it make sense to have an option to `filterForward` for setting the fallback addresses?

Reply:
pushed